nix/build-support/check-blueprints.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -o nounset -o pipefail -o errexit
+
+find src -name \*.blp -exec blueprint-compiler format {} \+
+find src -name \*.blp -exec blueprint-compiler compile --output=/dev/null {} \;
```

Comment:
For future note, I'm also totally fine just putting these directly in the GHA YAML files. I tend to only put scripts into `build-support` if a human can realistically run them regularly. In this case, I'm completely fine either way.